ignite_apps.md

```diff
@@ -0,0 +1,7 @@
+# Official Ignite Apps
+
+List of apps that will be installed by default:
+
+  - [network](github.com/ignite/cli-plugin-network/tree/v0.1.1): Launch a blockchain in production
+
+You can find other official apps in the [apps repository](https://github.com/ignite/apps).
```

Comment:
The repository URL might me different to the one here. It doesn't exist yet so I'm not sure of the final URL.

Reply:
> I'm not sure about use md file and not a `yaml` or `toml` file

Yes, it would be much better to parse them instead of Markdown, but I think that the idea was to actually use Markdown, maybe @ilgooz can confirm.

Reply:
Update: we should use YAML